src/python/pants/backend/docker/util_rules/docker_build_context.py

```diff
@@ -226,6 +226,12 @@ def create(
             version_context.get("build_args", {})
         )
 
+        # Data from Pants.
+        version_context["PANTS"] = {
+            # Present hash for all inputs that can be used for image tagging.
+            "HASH": str(hash((build_args, build_env, snapshot.digest))).replace("-", "0"),
+        }
```

Comment:
Not really sure if lowercase keys would be better, or lower.UPPER, or not nesting, so it would be `"...{pants_hash}.."`, etc.. 🤷🏽

Reply:
Opted for lowercasing, so it is `{pants.hash}`.